### PR TITLE
Added ping command

### PR DESCRIPTION
**Pull request by DoubleF3lix**

### CODE REVIEW

File: commanderbot_ext/ping/ping_cog.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+from discord.ext.commands import Bot, Cog, Context, command
+
+
+class PingCog(Cog):
+    def __init__(self, bot: Bot):
+        self.bot: Bot = bot
+        self._log: Logger = get_clogger(self)
+
+    @command(name="ping")
+    async def cmd_ping(self, ctx: discord.Message):
```
Comment (Arcensoth) [inline commanderbot_ext/ping/ping_cog.py:12]: `ctx` isn't a `Message`, it's an instance of `Context` from `discord.ext.commands`.

File: commanderbot_ext/ping/ping_cog.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import discord
+from commanderbot_lib.logging import Logger, get_clogger
+from discord.ext.commands import Bot, Cog, Context, command
+
+
+class PingCog(Cog):
+    def __init__(self, bot: Bot):
+        self.bot: Bot = bot
+        self._log: Logger = get_clogger(self)
```
Comment (Arcensoth) [inline commanderbot_ext/ping/ping_cog.py:9]: I don't see `self._log` actually being used anywhere here, so it should be removed along with the unused imports that follow.

File: commanderbot_ext/ping/ping_cog.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+    @command(name="ping")
+    async def cmd_ping(self, ctx: discord.Message):
+        await ctx.send(f"Pong! Latency: {round(self.bot.latency, 2)}ms")
```
Comment (Arcensoth) [inline commanderbot_ext/ping/ping_cog.py:13]: Side note: I'm on the fence about whether anything else should be in a simple ping command, but given that this is a [supported property](https://discordpy.readthedocs.io/en/latest/api.html#discord.Client.latency) from the discord.py API I think it's fine.

**Review by Arcensoth — CHANGES_REQUESTED** [review]

File: commanderbot_ext/ping/ping_cog.py (hunk 1 above)
Comment (DoubleF3lix) [reply commanderbot_ext/ping/ping_cog.py]: Fixed in latest commit

File: commanderbot_ext/ping/ping_cog.py (hunk 2 above)
Comment (DoubleF3lix) [reply commanderbot_ext/ping/ping_cog.py]: Fixed in latest commit

File: commanderbot_ext/ping/ping_cog.py (hunk 3 above)
Comment (DoubleF3lix) [reply commanderbot_ext/ping/ping_cog.py]: Fixed in latest commit (`ms` to `s`)

**Review by Arcensoth — APPROVED** [review]

**Pull request merged**